src/file.helpers.ts

```diff
@@ -1,16 +1,19 @@
+import { readFileSync } from 'fs';
+
 import { getProjectDirectoryPath } from './path.helpers';
 import { LxrConfig, PackageJson } from './interfaces';
 
+export function readJsonFile(path: string): unknown {
```

Comment:
Maybe we could define a generic parameter, so that the caller doesn't need to do type inference? Since `JSON.parse` returns `any`, this should be fine:

```ts
export function readJsonFile<T>(path: string): T {
 ...
}
```

Then, the caller can specify the type for the JSON file.